integration_tests/test_runner/src/bin/main.rs

```diff
@@ -155,6 +159,58 @@ pub async fn main() {
             info!("Start ICA Host test");
             ica_host_happy_path(&contact, &ibc_contact, keys, ibc_keys).await;
             return;
+        } else if test_type == "ONBOARDING_DEFAULT_PARAMS" {
+            start_ibc_relayer(&contact, &ibc_contact, &keys, &ibc_keys).await;
+            info!("Start onboarding default params test");
+            onboarding_default_params(
+                &contact,
+                &ibc_contact,
+                keys,
```

Comment:
there should be a human readable summary of the test goals somewhere, but maybe I'm missing it when looking here.